stock_picking_report_delivery_cost/models/stock_picking.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    currency_id = fields.Many2one(
+        related="sale_id.currency_id",
+        readonly=True,
+        string="Currency",
+        related_sudo=True,  # for avoiding access problems
+    )
```

Comment:
Why do you create this field if you don't use it later in the reports?

Reply:
`fields.Monetary` uses it automatically. I will put a comment about that.

Reply:
Ohh!! Ok, I do not know it